platformio.ini

```diff
@@ -242,6 +242,7 @@ build_flags =
     -DRED_PIN=01            ; GPIO to go HIGH on red color range
     -DBLUE_PIN=03           ; GPIO to go HIGH on orange color range
     -DSUPPORT_TFT
+    -DARDUINO_LILYGO_T_DISPLAY_S3
```

Comment:
Please rename define to LILYGO_T_DISPLAY_S3 to keep naming format as other boards